driver/Linker.ml

```diff
@@ -24,8 +24,8 @@ let linker exe_name files =
     ["-o"; exe_name];
     files;
     (if Configuration.has_runtime_lib
-     then ["-L" ^ !stdlib_path; "-lcompcert"]
-     else [])
+    then ["-L" ^ !stdlib_path; "-lcompcert"]
```

Comment:
It's good to avoid whitespace-only changes.  Plus, the previous indentation is better than the new one.